commands/createfixpullrequests.go

```diff
@@ -162,6 +165,22 @@ func (cfp *CreateFixPullRequestsCmd) createFixVersionsMap(params *utils.FrogbotP
 	return fixVersionsMap, nil
 }
 
+// getMinimalFixVersion that fixes the current impactedPackage
+// fixVersions array is sorted, so we take the first index, unless it's version is older than what we have now.
+func getMinimalFixVersion(impactedPackageVersion string, fixVersions []string) string {
+	var vulnFixVersion string
+	// Trim 'v' prefix in case of Go package
+	currVersion := strings.TrimPrefix(impactedPackageVersion, "v")
+	for index := range fixVersions {
+		fixVersionCandidate := parseVersionChangeString(fixVersions[index])
+		if version.NewVersion(currVersion).Compare(fixVersionCandidate) > 0 {
+			vulnFixVersion = fixVersionCandidate
+			break
+		}
+	}
+	return vulnFixVersion
```

Comment:
Please consider the following code:
```suggestion
	// Trim 'v' prefix in case of Go package
	currVersionStr := strings.TrimPrefix(impactedPackageVersion, "v")
	currVersion := version.NewVersion(currVersionStr)
	for _, fixVersion := range fixVersions {
		fixVersionCandidate := parseVersionChangeString(fixVersion)
		if currVersion.Compare(fixVersionCandidate) > 0 {
			return fixVersionCandidate
		}
	}
	return ""
```

Explanations:
1. version.NewVersion(currVersion) can be outside the loop
2. vulnFixVersion variable is redundant
3. More "go" style loop - read reference instead of index of range

Reply:
done